core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts

```diff
@@ -925,13 +926,24 @@ describe('Exploration editor page component', () => {
     }));
 
     it('should have component properties correspond to backend data', () => {
-      expect(component.explorationUrl).toBe('/create/' + explorationId);
-      expect(component.explorationDownloadUrl).toBe(
-        '/createhandler/download/' + explorationId
+      const interpolatedExplorationUrl = urlInterpolationService.interpolateUrl(
```

Comment:
In tests you can hardcode the expected value, please don't call UrlInterpolationService there. There should not be any logic in tests.